OpenRA.Mods.Common/Traits/Sound/AnnounceOnBuild.cs

```diff
@@ -19,6 +19,9 @@ public class AnnounceOnBuildInfo : ITraitInfo
 		[Desc("Voice to use when built/trained.")]
 		[VoiceReference] public readonly string Voice = "Build";
 
+		[Desc("Should the voice be played for the owner alone?")]
+		public readonly bool AnnounceOnlyToOwner = false;
```

Comment:
Considering we have some English-pickers in the form of @Phrohdoh I thought it's better to have it redundant than nonEnglish.